apps/explorer/src/components/module/module-functions-interaction/ModuleFunction.tsx

```diff
@@ -66,6 +69,14 @@ export function ModuleFunction({
             return result;
         },
     });
+    const executeFunctionError = execute.error
```

Comment:
I feel like you can just do the message extraction inline and avoid this variable that's super separated from the usage.